mne/viz/ica.py

```diff
@@ -267,9 +269,13 @@ def _plot_ica_sources_evoked(evoked, picks, exclude, title, show, labels=None):
 
     for exc_label, ii in zip(exclude_labels, picks):
         if exc_label is not None:
-            if labels is not None:
-                exc_label = exc_label.split(' - ')[1]
-            color = label_colors[exc_label]
+            # create look up for color ...
+            if ' - ' in exc_label:
```

Comment:
we don't need this second check any more, you also see when unwrapping a bit that this block follows an is None check. You see it?